ccp/src/cu/cu_prover.rs

```diff
@@ -23,7 +23,7 @@ use ccp_randomx::Dataset;
 use ccp_randomx::RandomXFlags;
 use ccp_shared::types::*;
 use ccp_utils::run_utils::run_unordered;
-use cpu_utils::CPUTopology;
+use cpu_utils::{CPUTopology, HwlocCPUTopology};
```

Comment:
Let's follow our coding standard and flatten them.

Reply:
Fixed